ipld/unixfs/mod/dagmodifier.go

```diff
@@ -46,6 +46,7 @@ type DagModifier struct {
 
 	Prefix    cid.Prefix
 	RawLeaves bool
+	MaxLinks  int
```

Comment:
@hsanjuan is this enough, given we've added separate setting for HAMT? (https://github.com/ipfs/boxo/pull/897#pullrequestreview-2726072733)

Reply:
It is probably enough since this seems to deal only with files...